misk/web/tabs/dashboard/webpack.config.js

```diff
@@ -29,7 +29,10 @@ module.exports = {
   output: {
     filename: '_admin/dashboard/tab_dashboard.js',
     path: path.join(__dirname, 'dist'),
-    publicPath: "/"
+    publicPath: "/",
+    library: ['Tabs', 'Dashboard'],
+    libraryTarget: 'umd',
+    globalObject: 'typeof self !== \'undefined\' ? self : this'
```

Comment:
This is crazy. What's happening here?

Reply:
- publicPath: url root that the package can be accessed at from webpack dev server and within the build structure if compiled with production flags
- library: library name, this one being in array means that it is a scoped package. Thus this would be accessible in the window by `Tabs.Dashboard`. The array names allow all tabs to be in a central location and prevent namespace issues if tabs end up being named the same as existing NPM packages that might be included in `@Misk/Common` and are bound as globals in window
- libraryTarget: type of module export `umd` is universal module description, options also include var if it's being bound to a variable and others
- globalObject: by default it tries to bind to window but this check prevents errors if it's ever included and run from Node or a non-browser environment. Without this, webpack complains during compilation.

Reply:
It's worth adding a note about the `globalObject` in particular.

Reply:
added to the next PR